src/cephadm/cephadm

```diff
@@ -2047,24 +2061,59 @@ def default_image(func: FuncT) -> FuncT:
     return cast(FuncT, _default_image)
 
 
+def get_container_info(ctx: CephadmContext, daemon_type_filter: str) -> Optional[ContainerInfo]:
+    """:return: Container inforamtion """
+    daemon_type, daemon_id = None, None
```

Comment:
Nit: but why not: `daemon_type = daemon_id = None`?

Reply:
code has changed with the latest changes but your comment is right :+1: